pkg/plugins/manifest_test.go

```diff
@@ -46,6 +47,11 @@ NR7DnB0CCQHO+4FlSPtXFTzNepoc+CytQyDAeOLMLmf2Tqhk2YShk+G/YlVX
 		require.NoError(t, err)
 		require.NotNil(t, manifest)
 		assert.Equal(t, "grafana-googlesheets-datasource", manifest.Plugin)
+		assert.Equal(t, "1.0.0-dev", manifest.Version)
+		assert.Equal(t, int64(1586817677115), manifest.Time)
+		assert.Equal(t, "7e4d0c6a708866e7", manifest.KeyID)
+		expectedFiles := []string{"LICENSE", "README.md", "gfx_sheets_darwin_amd64", "gfx_sheets_linux_amd64", "gfx_sheets_windows_amd64.exe", "module.js", "module.js.LICENSE.txt", "module.js.map", "plugin.json"}
```

Comment:
It's more readable if you write a long array over several lines:
```suggestion
		expectedFiles := []string{
			"LICENSE", "README.md", "gfx_sheets_darwin_amd64", "gfx_sheets_linux_amd64",
			 "gfx_sheets_windows_amd64.exe", "module.js", "module.js.LICENSE.txt", "module.js.map",
			 "plugin.json",
		}
```